lesson-0-open-house/README.md

```diff
@@ -0,0 +1,98 @@
+# Workshop pilot
+
+## Recipe
+
+### Part 1
+
+0. Welcome everyone and introduce yourself
+1. Introduce the workshop
+   * Not OpenShift operations or provisioning
+2. Show location of materials
+3. Introduce OpenShift (short)
+4. [Developer Sandbox - try OpenShift yourself easily](https://developers.redhat.com/developer-sandbox/get-started).
+4. Talk about the difference between OpenShift and Kubernetes
+5. [Open the demo](http://websockets-demo-ttomecek-dev.apps.sandbox.x8i5.p1.openshiftapps.com/), show URL so that everyone can open it too.
+   * Click and fool around
+
+### Part 2
+
+1. [Open webconsole](https://console-openshift-console.apps.sandbox.x8i5.p1.openshiftapps.com/topology/ns/ttomecek-dev?view=graph)
+   * Show the project (skip most of the webconsole elements - that's the course)
+   * Briefly describe pod/deployment, service, route
+   * Show metrics/logs
+2. Showcase `oc` binary
+   * "Copy login command"
+   * status, get pods, describe, logs
+   * login, whoami - explain the authentication mechanism
+
+## Demo project
+
+You can try it locally:
+```
+$ podman run -p 8081:8081 quay.io/tomastomecek/websockets-demo:latest
+```
+
+And then open [0.0.0.0:8081](https://0.0.0.0:8081/) in your browser.
+
+
+## Workshop steps
+
+Once you get your Sandbox cluster provisioned, get the login command and invoke it in your shell.
+
+NEVER SHARE YOUR API TOKEN
+
+```
+$ oc login --token=sha256~$NUMBERS_LETTERS_UNICORNS --server=https://api.sandbox.x8i5.p1.openshiftapps.com:6443
+```
+
+### Get kubeconfig
+
+Do we have access?
+```
+$ oc status
+In project ttomecek-dev on server https://api.sandbox.x8i5.p1.openshiftapps.com:6443
+
+You have no services, deployment configs, or build configs.
+Run 'oc new-app' to create an application.
+
+$ oc whoami
+ttomecek
```

Comment:
fun fact: `oc get users '~' -oyaml` gives you more human-readable details (but the meaning of `~` is ofc harder to understand to a normal human)

Reply:
whaaaaat

Reply:
does the tilde imply me? because I don't have perms to do `oc get users` but your command works well

Reply:
The tilde is indeed the user echo endpoint. If you did `oc whoami --loglevel 10`, you would see that this user API endpoint is actually being hit and you would see the same object being returned in the logs :slightly_smiling_face: